tests/neo-vm.Tests/Tests/OpCodes/Slot/INITSLOT.json

```diff
@@ -0,0 +1,125 @@
+{
+  "category": "Slot",
+  "name": "INITSLOT",
+  "tests": [
+    {
+      "name": "Real test [LocalVariables]",
+      "script": [
+        "INITSLOT",
+        "0x0100"
```

Comment:
What does it mean @shargon ? INITSLOT accepts a number in single byte or double byte formats? How many slots being created here? Is it 01 and the other 00 is opcode PUSH0? Or is it 256 (BE) or 01 (LE) ?

Reply:
Read the code, it's easy to be understood.